.eslintrc

```diff
@@ -20,10 +21,41 @@
     "import/prefer-default-export": "off",
     "camelcase": "off"
   },
-  "overrides": [{
-    "files": ["prelude/**/*"],
-    "rules": {
-      "strict": "off"
+  "overrides": [
+    {
+      "files": ["prelude/**/*"],
+      "rules": {
+        "strict": "off"
+      },
+    }   
+    , {
+      "files": ["test/**/*"],
+      "rules": {
+        "array-callback-return": "off",
+        "func-names": "off",
+        "global-require": "off",
+        "guard-for-in": "off",
+        "import/extensions": "off",
+        "import/no-dynamic-require": "off",
+        "import/no-extraneous-dependencies": "off",
+        "import/newline-after-import": "off",
+        "import/no-unresolved": "off",
+        "import/no-useless-path-segments": "off",
+        "import/order": "off",
+        "no-console": "off",
+        "no-lonely-if": "off",
+        "no-multi-assign": "off",
+        "no-undef": "off",
```

Comment:
Could we remove some rules here?

Reply:
the PR will be massive so many tests file will need updating. Are you ready ?

Reply:
We could just take this PR as is and have follow up PRs that removes flags (I could help)

Reply:
Ok for me, just asking :) Typescript is good but if we disable strict and other rules it becomes almost useless :laughing:

Reply:
We could disable one by one